examples/openvino/question-answering/README.md

```diff
@@ -44,4 +44,33 @@ python run_qa.py \
   --max_seq_length 384 \
   --doc_stride 128 \
   --output_dir /tmp/outputs_squad/
-```
\ No newline at end of file
+```
+
+### Joint Pruning, Quantization and Distillation (JPQD) for BERT on SQuAD1.0
+`OVTrainer` also provides an advanced optimization workflow through the NNCF when Transformer model can be structurally pruned along with 8-bit quantization and distillation. Below is an example which demonstrates how to jointly prune, quantize BERT-base for SQuAD 1.0 using NNCF config `--nncf_compression_config` and distill from BERT-large teacher. This example closely resembles the movement sparsification work of [Lagunas et al., 2021, Block Pruning For Faster Transformers](https://arxiv.org/pdf/2109.04838.pdf). This example takes about 12 hours with a single V100 GPU and the output model attains about 40% structured sparsity (denominated by linear layers only). The joint pruned and quantized OpenVINO IR provides >60% more throughput over quantize-only model on AWS EC2 instance (c6i.32xlarge). Attributed to large teacher distillation, the sparse-quantized model has F1 improved to >89%.
```

Comment:
I would prefer to add a link towards an existing benchmark, removing this part in the meantime.
```suggestion
`OVTrainer` also provides an advanced optimization workflow through the NNCF when Transformer model can be structurally pruned along with 8-bit quantization and distillation. Below is an example which demonstrates how to jointly prune, quantize BERT-base for SQuAD 1.0 using NNCF config `--nncf_compression_config` and distill from BERT-large teacher. This example closely resembles the movement sparsification work of [Lagunas et al., 2021, Block Pruning For Faster Transformers](https://arxiv.org/pdf/2109.04838.pdf). This example takes about 12 hours with a single V100 GPU and the output model attains about 40% structured sparsity (denominated by linear layers only). 
```
Also concerning the  40% structured sparsity,I think we should make it clearer that it's on the model without taking into account the embeddings

Reply:
We do mention it "about 40% structured sparsity _(denominated by linear layers only)_"